independent-publisher-connectors/Google Photos/apiDefinition.swagger.json

```diff
@@ -0,0 +1,161 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",

```

Comment:
@juliamuiruri4 please change the version to "1.0"

Reply:
Done